PushNotifications.md

```diff
@@ -240,7 +240,7 @@ final message = ably.Message(
 
 ### Receiving Messages
 
-For examples of handling incoming messages and dealing with notifications, see [push_notification_handlers](example/lib/push_notifications/push_notification_handlers.dart) in the example app.
+For examples of handling incoming messages and dealing with notifications, see [push_notification_handlers](example/lib/push_notifications/push_notification_handlers.dart) in the example app. There are 2 types of push notifications, one which shows as a notification to the user (alert push notification on iOS, notification message on Android), and one which notifies your app (background notification on iOS, data message on Android) 
 
```

Comment:
I think it would be nice to separate those two on separate two bullet points.

Reply:
Yup, it's nicer: see a32b6ae3a3f199676d0fdb1a4cdcc6926e07ffc8

Reply:
I'm closing this conversation on @ikbalkaya's behalf.